kart/log.py

```diff
@@ -51,19 +85,62 @@
 @click.argument("args", nargs=-1, type=click.UNPROCESSED)
 def log(ctx, output_format, json_style, do_dataset_changes, with_feature_count, args):
     """ Show commit logs """
+    repo = ctx.obj.get_repo(allowed_states=KartRepoState.ALL_STATES)
+
+    # Handle positional args.
```

Comment:
Arg parsing could probably have it's own function. No harm in adding more functions, makes code easier to read.